### ISSUE TITLE
Not able to choose different fanart

### ISSUE THREAD

**Issue by Desperados85** [henryjfry/script.artwork.beef-matrix#2]
Hi! 

I've been using Artwork Beef with Kodi Leia without issues and last night I finally switched to Matrix because I read about your Matrix version! 

I can't really scrape my artwork with the default Kodi scrapers because there's no option to limit the amount of artwork it scrapes. My limited storage on the Shield always runs low when scraping artwork with TMDb.

So I completely wiped my install and started building my library from scratch. (I have setup a virtual library using TMDb Helper). After that I installed Artwork Beef, set every artwork type to 'one' (like I did in the Leia version) and started scraping. After it was done I checked my library and found that a lot of fanart had titles in them! 
I double checked the settings and 'Prefer fanart with no title' was enabled. 

So I didn't understand the problem until I read somewhere that you kind of combined the fanart and landscape images. I thought no big deal I'll just switch the fanart manually, so I clicked the context menu; manage; AB: Select artwork. It then shows the menu with all the different artwork, it says 'x amount' available, but when I click on fanart, nothing shows and I can't select anything. 

(See screenshots below) 

So I thought maybe if I disable the language in the settings, change the amount of scraped artwork to 'three' and try again.. So I deleted all the artwork and scraped again.. But still the same problem: Titles in fanart and no way to manually change the artwork. 
I also noticed I didn't like some artwork that came from fanart.tv so I set the preferred scraper to TMDb, which is only an option for movies, would be great if that option would be available for tv shows as well. 

Another question is regarding the rating of the scraped artwork, from 0-10, what does that mean? Cause I can't see ratings on the sites, I went to fanart.tv and TMDb.. Obviously I want the best artwork for my media and noticed if I set the rating to 10 almost nothing gets scraped, but if I set to 0 I get pretty bad fanart while my size is set to 1920x1080.. I've been at it for hours trying to built my library and finally gave up so I thought maybe I should open an issue on GitHub lol. 

Hope you can help me! 

![Kodi_20211204_115057](https://user-images.githubusercontent.com/46796194/144706935-5858d0dd-bf34-45e7-a80d-b941127b57e1.png)
![Kodi_20211204_115149](https://user-images.githubusercontent.com/46796194/144706937-0cc58d8b-8846-4211-bdc6-d05a81377869.png)
![Kodi_20211204_115216](https://user-images.githubusercontent.com/46796194/144706938-bc7e133c-1134-4def-8579-0be13d92bd2e.png)

**Comment by henryjfry** [comment]
Ok I had a look at the show Dark on my setup with Artwork beef, and I got artwork showing up at my end.

I made the change to artwork beef to use fanart/landscape art for shows where no fanart/landscape is actually returned.  I'd much rather have some slighty "wrong" art than none at all.

So yes this would no longer respect the "titles" setup, which is pretty hit and miss whether or not the original fanart uploader acutally set it up correctly.

Without full logs im not sure if I can help you.  Even with full logs im not sure I can help you, i ported it over to kodi 19 and made some changes but there is a bunch of stuff in the addon im not 100% sure how it works.
Ratings and stuff, im not sure about either.  I dont even know how you rate stuff on fanart tv so again I wouldnt trust ratings at all.

So adding an additional scraper for TV shows, probably not going to happen.

Basically my experience with kodi fanart is that you will probably need to manually fix a bunch of stuff and just plug the gaps in your library artwork yourself with artwork beef/kodis builtin as a jumping off point
